test/features/install_trusty.feature

```diff
@@ -0,0 +1,71 @@
+@trusty
+@infinispan
+@kafka
+Feature: Kogito Trusty
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed with Infinispan and Kafka operators
+
+  @smoke
+  Scenario: Install Kogito Trusty
+    When Install Kogito Trusty with 1 replicas
+    Then Kogito Trusty has 1 pods running within 10 minutes
+    
+#####
+
+  @externalcomponent
+  @infinispan
+  @kafka
+  Scenario: Install Kogito Trusty with persistence using external Infinispan
+    Given Infinispan instance "external-infinispan" is deployed with configuration:
+      | username | developer |
+      | password | mypass |
+    And Kafka instance "external-kafka" is deployed
+    When Install Kogito Trusty with 1 replicas with configuration:
+      | infinispan | username | developer                 |
+      | infinispan | password | mypass                    |
+      | infinispan | uri      | external-infinispan:11222 |
+      | kafka | externalURI | external-kafka-kafka-bootstrap:9092 |
+    Then Kogito Trusty has 1 pods running within 10 minutes
+
+
+#####
+
+  @events
+  @kafka
+  @infinispan
+  Scenario: Trusty retrieves tracing events using external Kafka
+    Given Kogito Operator is deployed with Infinispan and Kafka operators
+    And Kafka instance "external-kafka" is deployed
+    And Infinispan instance "external-infinispan" is deployed with configuration:
+      | username | developer |
+      | password | mypass |
+    And Install Kogito Trusty with 1 replicas with configuration:
+      | infinispan | username | developer                 |
+      | infinispan | password | mypass                    |
+      | infinispan | uri      | external-infinispan:11222 |
+      | kafka | externalURI | external-kafka-kafka-bootstrap:9092 |
+    And Local example service "dmn-tracing-quarkus" is built by Maven using profile "default" and deployed to runtime registry
```

Comment:
Problem in PR check and/or later in nightly is that the `dmn-tracing-quarkus` will be never built in pipeline and thus failing in the end as cannot start ...
you should also add the build of runtime image in https://github.com/kiegroup/kogito-cloud-operator/tree/master/test/scripts/examples